gems/omniauth-saml/GHSA-cvp8-5r8g-fhvq.yml

```diff
@@ -0,0 +1,23 @@
+---
+gem: omniauth-saml
+ghsa: cvp8-5r8g-fhvq
+url: https://github.com/SAML-Toolkits/ruby-saml/security/advisories/GHSA-jw9c-mfg7-9rx2
+title: omniauth-saml vulnerable to Improper Verification of Cryptographic Signature
+date: 2024-09-11
+description: "ruby-saml, the dependent SAML gem of omniauth-saml has a signature wrapping
```

Comment:
Please use `description: |` so that the text isn't an escaped double-quoted String, which makes it easier to read and edit.